pyfftw/interfaces/__init__.py

```diff
@@ -116,6 +116,23 @@
 * :func:`pyfftw.interfaces.numpy_fft.hfft`
 * :func:`pyfftw.interfaces.numpy_fft.ihfft`
 
+:mod:`~pyfftw.interfaces.scipy_fft`
+"""""""""""""""""""""""""""""""""""""""
+
+* :func:`pyfftw.interfaces.scipy_fft.fft`
+* :func:`pyfftw.interfaces.scipy_fft.ifft`
+* :func:`pyfftw.interfaces.scipy_fft.fft2`
+* :func:`pyfftw.interfaces.scipy_fft.ifft2`
+* :func:`pyfftw.interfaces.scipy_fft.fftn`
+* :func:`pyfftw.interfaces.scipy_fft.ifftn`
+* :func:`pyfftw.interfaces.scipy_fft.rfft`
+* :func:`pyfftw.interfaces.scipy_fft.irfft`
+* :func:`pyfftw.interfaces.scipy_fft.rfft2`
+* :func:`pyfftw.interfaces.scipy_fft.irfft2`
+* :func:`pyfftw.interfaces.scipy_fft.rfftn`
+* :func:`pyfftw.interfaces.scipy_fft.irfftn`
+* :func:`pyfftw.interfaces.scipy_fft.next_fast_len`
```

Comment:
It looks like `hfft` and `ihfft` are missing from this list

Reply:
Thanks, fixed